DOC-258 Data Lake Dedupe (#1878)
segmentio/segment-docs: src/connections/storage/data-lakes/index.md

![](images/dl_vpc.png)

Data Lakes offers 12 syncs in a 24 hour period and doesn't offer a custom sync schedule or selective sync.

### Data Lake deduplication

> In addition to Segment's [99% guarantee of no duplicates](/docs/guides/duplicate-data/) for data within a 24 hour look-back window, Data Lakes have another layer of deduplication to ensure clean data in your Data Lake. Segment removes duplicate events at the time your Data Lake ingests data. The Data Lake deduplication process dedupes the data the Data Lake syncs within the last 7 days, with Segment deduping the data based on the `message_id`.

Reviewer: @stayseesong The last sentence here is a bit hard to follow. Might just be all the instances of the word 'data'.

Maybe something like this?

```suggestion
In addition to Segment's [99% guarantee of no duplicates](/docs/guides/duplicate-data/) for data within a 24 hour look-back window, Data Lakes have another layer of deduplication to ensure clean data in your Data Lake. Segment removes duplicate events at the time your Data Lake ingests data.  Data Lakes deduplicate any data synced within the last 7 days, based on the `message_id` field.
```
